tests/integrationv2/test_well_known_endpoints.py

```diff
@@ -84,11 +84,17 @@
             {"cipher": "ECDHE-RSA-AES256-GCM-SHA384", "kem": "NONE"},
     }
 
+if get_flag(S2N_USE_CRITERION) != "off":
+    provider = [CriterionS2N]
+    timeout = 60
+else:
```

Comment:
Why is the timeout longer for Criterion? Could we handle the difference in managed_process instead of manually adding it to every test?

Similar question for the provider change: it doesn't seem great to have to add this logic to every existing and new test in order to support Criterion.